webpack.config.js

```diff
@@ -56,13 +56,13 @@ module.exports = {
           new CopyWebpackPlugin({
             patterns: [
               {
-                from: path.resolve(__dirname, "./static/**/*.svg"),
-                to: "./"
-              }
-            ]
-          })
+                from: path.resolve(__dirname, "./static"),
+                to: "./",
```

Comment:
i think i might accidentally grab the index.html too, wanna do the `.(svg|jpeg|jpg|png)` instead?

Reply:
oo that's a good catch!